ister_test.py

```diff
@@ -116,7 +116,7 @@ def run_command_wrapper(func):
     def wrapper():
         """run_command_wrapper"""
         def mock_run_command(cmd, _=None, raise_exception=True,
-                             log_output=True):
+                             log_output=True, environ=None):
```

Comment:
So the environ should be added to the COMMAND_RESULTS list when not none in order to validate it is actually getting the expected environ content.